crates/atomic/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Dawid Ciężarkiewicz <dpc@dpc.pw>"]
 description = "Atomic run-time controllable drain for slog-rs"
 keywords = ["slog", "logging", "log",  "atomic"]
 license = "MPL-2.0"
-documentation = "https://dpc.github.io/slog-rs/"
+documentation = "https://docs.rs/slog-atomic"
```

Comment:
You're English is good, no worries. You're right. Sorry for the trouble, it's just  the slog was recently split into more repos and moved to own github community to accommodate growing team. I'll look into it.